src/BitTorrent-1.1.xml

```diff
@@ -0,0 +1,509 @@
+             with, or affiliated with, you. For purposes of this definition, control means (i) the power,
+             direct or indirect, to cause the direction or management of such entity, whether by contract
+             or otherwise, or (ii) ownership of fifty percent (50%) or more of the outstanding shares, or
+             (iii) beneficial ownership of such entity. You are responsible for advising any affiliated
+             entity of the terms of this License, and that any rights or privileges derived from or
+             obtained by way of this License are subject to the restrictions outlined herein.</p>
+        </li>
+        <li>
+          <b>15.</b>
+          <p>Glossary. All defined terms in this License that are used in more than one Section of this
+             License are repeated here, in alphabetical order, for the convenience of the reader. The
+             Section of this License in which each defined term is first used is shown in parentheses.</p>
+        </li>
+      </list>
+      <p>Contributor: Each person or entity who created or contributed to the creation of, and distributed, a
+         Modification. (See Section 2)</p>
+      <p>Derivative Works: That term as used in this License is defined under U.S. copyright law. (See Section 1(b))</p>
+      <p>License: This BitTorrent Open Source License. (See first paragraph of License)</p>
+      <p>Licensed Product: Any BitTorrent Product licensed pursuant to this License. The term &quot;Licensed
+         Product&quot; includes all previous Modifications from any Contributor that you receive. (See first
+         paragraph of License and Section 2)</p>
+      <p>Licensor: BitTorrent, Inc. (See first paragraph of License)</p>
+      <p>Modifications: Any additions to or deletions from the substance or structure of (i) a file containing Licensed 
+        <br/>Product, or (ii) any new file that contains any part of Licensed Product. (See Section 2) 
+      </p>
+      <p>Notice: The notice contained in Exhibit A. (See Section 4(e))</p>
+      <p>Source Code: The preferred form for making modifications to the Licensed Product, including all modules
+         contained therein, plus any associated interface definition files, scripts used to control compilation
+         and installation of an executable program, or a list of differential comparisons against the Source
+         Code of the Licensed Product. (See Section 1(a))</p>
+      <p>You: This term is defined in Section 14 of this License.</p>
+    </body>
+    <optional>
+      <p>EXHIBIT A</p>
+      <p>The Notice below must appear in each file of the Source Code of any copy you distribute of the Licensed
+         Product or any hereto. Contributors to any Modifications may add their own copyright notices to
+         identify their own contributions.</p>
+      <p>License: 
+        <br/>The contents of this file are subject to the BitTorrent Open Source License Version 1.0 (the
```

Comment:
this is a standard header, but I don't see the header tag included anywhere - need to add, I think